libraries/AC_Fence/AC_Fence.cpp

```diff
@@ -42,20 +42,24 @@ extern const AP_HAL::HAL& hal;
 
 
 const AP_Param::GroupInfo AC_Fence::var_info[] = {
+
+    // 0 - deprecated enable flag
+
     // @Param: ENABLE
     // @DisplayName: Fence enable/disable
-    // @Description: Allows you to enable (1) or disable (0) the fence functionality
-    // @Values: 0:Disabled,1:Enabled
+    // @Description: Enabled fence types held as bitmask
+    // @Bitmask{Rover}: 1:Circle Centered on Home,2:Inclusion/Exclusion Circles+Polygons
+    // @Bitmask{Copter, Plane, Sub}: 0:Max altitude,1:Circle Centered on Home,2:Inclusion/Exclusion Circles+Polygons,3:Min altitude
     // @User: Standard
-    AP_GROUPINFO("ENABLE",      0,  AC_Fence,   _enabled,   0),
+    AP_GROUPINFO("ENABLE",      12,  AC_Fence,   _enabled_fences,   0),
```

Comment:
I think it is dangerous to change FENCE_ENABLE to a bitmask as users may load old parameters and end up with only the max alt fence when they set FENCE_ENABLE=1

Reply:
Fixed thanks